walletlibrary/src/main/java/com/microsoft/walletlibrary/did/sdk/util/controlflow/Result.kt

```diff
@@ -23,10 +25,16 @@ internal fun <T> KotlinResult<T>.toSDK(): Result<T> {
         }
     }
     this.exceptionOrNull()?.let {
-        if (it is SdkException) {
-            return Result.Failure(it)
-        } else {
-            return Result.Failure(SdkException("Could not cast failure to SDK", it.cause))
+        when (it) {
+            is SdkException -> {
+                return Result.Failure(it)
+            }
+            is NetworkingException -> {
+                return Result.Failure(SdkException("Networking Exception", it))
+            }
+            else -> {
+                return Result.Failure(SdkException("Could not cast failure to SDK", it.cause))
```

Comment:
is It possible to change the string "Could not cast failure to SDK" to be like the message that describes the failure or something?